test/other.jl

```diff
@@ -0,0 +1,7 @@
+using Metalhead, Test
+using Flux
+
+@testset "MLPMixer" begin
+    @test size(MLPMixer()(rand(Float32, 256, 256, 3, 67))) == (1000, 67)
+    @test_skip gradtest(MLPMixer(), rand(Float32, 256, 256, 3, 67))
```

Comment:
The batch size needs to be small for CI to run.

```suggestion
    @test size(MLPMixer()(rand(Float32, 256, 256, 3, 2))) == (1000, 2)
    @test_skip gradtest(MLPMixer(), rand(Float32, 256, 256, 3, 2))
```